tta-server/app/models/Population.scala

```diff
@@ -0,0 +1,24 @@
+package models
+
+object Population {
+
+  def generateIncreasePopulationAction(playerState: PlayerState): Action = new Action{
```

Comment:
this does not need a `playerState`, does it?

also, space between `Action` and `{`